config/ext_jenkins-weekly.yaml

```
@@ -0,0 +1,176 @@
+serviceAccount:
+  create: true
+  name: jenkins-controller
+serviceAccountAgent:
+  create: true
+  name: jenkins-agent
+rbac:
+  create: true
+  readSecrets: true
+persistence:
+  enabled: true
+  size: 50Gi
+agent:
+  componentName: "agent"
+networkPolicy:
+  enabled: true
+  internalAgents:
+    allowed: true
+    namespaceLabels:
+      name: "jenkins-weekly"
+controller:
+  image: jenkinsciinfra/jenkins-weekly
+  tag: 0.50.0-2.341
+  pullPolicy: IfNotPresent
+  resources:
+    limits:
+      cpu: "2"
+      memory: "4Gi"
+    requests:
+      cpu: "2"
+      memory: "4Gi"
+  probes:
+    startupProbe:
+      initialDelaySeconds: 120
+    livenessProbe:
+      initialDelaySeconds: 120
+    readinessProbe:
+      initialDelaySeconds: 120
+  testEnabled: false
+  podSecurityContextOverride:
+    runAsNonRoot: true
+    runAsUser: 1000
+    supplementalGroups: [1000]
+  overwritePlugins: true
+  serviceType: "ClusterIP"
+  javaOpts: >
+    -XshowSettings:vm -XX:+UseStringDeduplication -XX:+ParallelRefProcEnabled -XX:+DisableExplicitGC -XX:MaxRAM=4g -XX:+AlwaysPreTouch -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath=/tmp/ -XX:+UseG1GC -Dorg.csanchez.jenkins.plugins.kubernetes.pipeline.ContainerExecDecorator.websocketConnectionTimeout=60
+
+  JCasC:
+    enabled: true
+    defaultConfig: false
+    configScripts:
+      security: |
+        security:
+          scriptApproval:
+            approvedSignatures:
+              - "method org.jenkinsci.plugins.workflow.steps.FlowInterruptedException getCauses"
+              - "new java.lang.RuntimeException java.lang.Throwable"
+      credentials: |
+        credentials:
+          system:
+            domainCredentials:
+              - credentials:
+                - gitHubApp:
+                    appID: "${GITHUB_APP_ID}"
+                    description: "GitHub App for weekly.ci.jenkins.io"
+                    id: "github-app-weekly"
+                    privateKey: "${GITHUB_APP_PRIVATE_KEY}"
+                    scope: GLOBAL
+      agent-settings: |
+        jenkins:
+          numExecutors: 1 # Allow one build on controller for now
```

```suggestion
          numExecutors: 0 # Allow one build on controller for now
```

Absolutely no executor on the controller 😅 that would allow reading the credentials (including LDAP)